### DISCUSSION TITLE
Performance

### DISCUSSION THREAD

**Discussion by sysoev-dev** [AndresGarciaSobrado91/MSI-Modern15-Hackintosh#22] [Q&A] [answered]
Hello! I have MSI Modern 15 a10m installed on it Manjaro Sway, what are the advantages of OS x versus linux?  
Or is it just taste?)

P.S I really like this kind of concept. I've tried Fedora Gnome, but I've experienced a number of performance issues.

**Comment by AndresGarciaSobrado91** [comment, marked as answer]
Hi! In terms of advantages, I think you will have the same tools that you have in linux but with an awesome UI. With macOS you are getting the best of two worlds (windows-linux). You should give it a try.
in terms of performance, the hackintosh is running smoothly as if it were running in a real macbook 👍

**Comment by sysoev-dev** [comment]
Sorry for posting my discussion in the wrong place. Thank you very much for your reply! According to the OpenCore instructions, I will try to install Mac OS on my MSI modern 15 a10m
